ddtrace/tracer/span.go

```diff
@@ -284,8 +311,7 @@ func (s *span) setMetric(key string, v float64) {
 	switch key {
 	case ext.SamplingPriority:
 		// setting sampling priority per spec
-		s.Metrics[keySamplingPriority] = v
-		s.context.setSamplingPriority(int(v))
+		s.setSamplingPriorityLocked(int(v), samplerManual, 0)
```

Comment:
Is there any reason to keep this logic? I think the customer should use `ext.ManualDrop` or `ext.ManualKeep` instead of `ext.SamplingPriority`.

Reply:
Can be `samplerUnknown`